packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { PaymentConsentCheckbox } from './index';
+import { Form } from '../fields';
+import useValidatorState from '../../lib/validator';
+import { MockPage } from '../../../.storybook/components/MockApp';
+import { SELECTED_PLAN } from '../../lib/mock-data';
+
+const WrapCheckbox = () => {
+  const validator = useValidatorState();
+  return (
+    <Form validator={validator}>
```

Comment:
TIL: We need to wrap the checkbox in a `Form`, because it gets the `validator` object from the form's context.